stl/inc/yvals_core.h

```diff
@@ -1140,7 +1141,31 @@
 #define _CXX20_DEPRECATE_IS_ALWAYS_EQUAL
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4034
+#if _HAS_CXX23 && !defined(_SILENCE_CXX23_ALIGNED_STORAGE_DEPRECATION_WARNING) \
+    && !defined(_SILENCE_ALL_CXX23_DEPRECATION_WARNINGS)
+#define _CXX23_DEPRECATE_ALIGNED_STORAGE                                              \
+    [[deprecated("warning STL4034: "                                                  \
+                 "std::aligned_storage is deprecated in C++23 by P1413R3. "           \
+                 "Prefer alignas(T) std::byte t_buff[sizeof(T)]."                     \

```

Comment:
There's a missing space after the "Prefer" sentence, hidden by the line break. Occurs in the next message.